test/ApplicationInsightsJavaScriptSnippetTest/ApplicationInsightsJavaScriptSnippetTest.cs

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Server.IntegrationTesting;
+using Microsoft.AspNetCore.Server.IntegrationTesting.xunit;
+using Microsoft.AspNetCore.Testing.xunit;
+using Microsoft.Extensions.Logging;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace ApplicationInsightsJavaScriptSnippetTest
+{
+    public class ApplicationInsightsJavaScriptSnippetTest : LoggedTest
+    {
+        public ApplicationInsightsJavaScriptSnippetTest(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [ConditionalTheory]
+        [OSSkipCondition(OperatingSystems.Linux)]
+        [OSSkipCondition(OperatingSystems.MacOSX)]
+        [InlineData(ApplicationType.Portable)]
+        [InlineData(ApplicationType.Standalone)]
+        public async Task ApplicationInsightsJavaScriptSnippetTest_OnWindowsOS(ApplicationType applicationType)
+        {
+            await JavaScriptSnippetInjectionTestSuite(applicationType);
+        }
+
+        [ConditionalTheory]
+        [OSSkipCondition(OperatingSystems.Windows)]
+        [InlineData(ApplicationType.Portable)]
+        [InlineData(ApplicationType.Standalone)]
+        public async Task ApplicationInsightsJavaScriptSnippetTest_OnNonWindows(ApplicationType applicationType)
```

Comment:
Remove the prefix `ApplicationInsightsJavaScriptSnippetTest_` as the full name of the test would be 'namespace.classname.methodname'